go.mod

```diff
@@ -3,6 +3,10 @@ module github.com/grafana/xk6-amqp
 go 1.15
 
 require (
+	github.com/Shimmur/proto_schemas_go v0.128.0
+	github.com/golang/protobuf v1.5.0
```

Comment:
Not a blocker, but apparently google recommends using: "google.golang.org/protobuf". There might be API changes here though.

Reply:
Ya our library is not compatible with that one, I tried :(. @talpert confirmed this as well because I got a warning from it.